src/test/java/io/swagger/oas/test/BodyParamExtractionTest.java

```diff
@@ -66,7 +66,7 @@ public void testStringBodyParam() throws Exception {
                         .addMediaType("application/json",new MediaType()
                             .schema(new Schema().type("string"))));
 
-        JavaType jt = utils.getTypeFromRequestBody(body, definitions);
+        JavaType jt = utils.getTypeFromRequestBody(body, definitions)[0];
```

Comment:
if the array is null or empty, this will crash.

Reply:
I'm sure it will not be null on the test

Reply:
keep in mind that inside method `utils.getTypeFromRequestBody` https://github.com/swagger-api/swagger-inflector/blob/form-data-parameters/src/main/java/io/swagger/oas/inflector/utils/ReflectionUtils.java#L144

there is a chance that a null value is returned. if this will never happen you should remove the `if-check`. Or throw an exception when the `if-check` is not matched, that way some one else will know why there is something wrong.